docker-compose.yml

```diff
@@ -197,5 +200,11 @@ services:
                     - mongo-useradm
                     - mongo-workflows
 
+    mender-nats:
+        image: nats:2.6.2-alpine3.14
```

Comment:
Latest version of NATS is 2.6.4.
```suggestion
        image: nats:2.6.4-alpine3.14
```

Reply:
With mongo, we secify the minor version (`4.4`); should we pin `2.6` for nats, maybe?